doc/developer/mesh.md

```diff
@@ -109,6 +109,14 @@ During manipulation, if two wedges become the same (i.e. for the example on the
 To merge wedges an explicit call to TopologicalMesh::mergeEqualWedges is needed.
 When a wedge is not referenced anymore (because referencing halfedges have been deleted), it is marked for deletation. When one call Ra::Core::Geometry::TopologicalMesh::garbage_collection(), the marked wedges are eventually deleted and halfedges index are updated accordingly.
 
+During convertion from Core::Geometry::*Mesh to Core::Geometry::TopologicalMesh, non manifold mesh faces are not added but given to a optionally provided user defined functor (\see Ra::Core::Geometry::TopologicalMesh::TopologicalMesh).
+Some meshes have multiple times the **same** vertices with **different** values for attributes inside a single face.
```

Comment:
And ? What is the bad effect of such a case? Given a cube, vertices have the same position but with 3 attributes (one par square face). And this situation is not problematic as wedges are built for that, right ?

Reply:
inside a single face

Reply:
```suggestion
Some meshes have multiple times the **same** vertices with **different** values for attributes inside a **single** face.
```

Reply:
Also there is a figure to show the problem, it's really related to bad topology, for instance if the top face of the cube is a pentagon with two time the same position with different normals. The rendering will change dramatically depending on triangulation, and it's bad ;)